[BUG] generated package.json has double encoded strings

**Describe the bug**
```
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name ""^4.2.0"": Tags may not have any characters that encodeURIComponent encodes.```
```
**Expected behavior**
`npm i` to return without errors

**To Reproduce**
create an application package

**test**
adjust https://github.com/eXist-db/generator-exist/blob/361055d6e6949e33b4e8f8c0453e8ecf4b686d4b/test/util/consistency.js#L100 

tweak?
```javascript
var pattern = /^\^?\d+\.+/
```

**Context (please always complete the following information):**
 -  App Version: `HEAD` master branch

--- a/generators/app/index.js
+++ b/generators/app/index.js
@@ -414,13 +414,13 @@ module.exports = class extends Generator {
       bugs: '',
       keywords: ['exist', 'exist-db', 'xml', 'xql', 'xquery'],
       devDependencies: {
-        chai: JSON.stringify(pjson.devDependencies.chai),
-        'chai-xml': JSON.stringify(pjson.devDependencies['chai-xml']),
-        'fs-extra': JSON.stringify(pjson.devDependencies['fs-extra']),
-        mocha: JSON.stringify(pjson.devDependencies.mocha),
-        supertest: JSON.stringify(pjson.devDependencies.supertest),
-        xmldoc: JSON.stringify(pjson.devDependencies.xmldoc),
-        'yeoman-assert': JSON.stringify(pjson.devDependencies['yeoman-assert'])
+        chai: pjson.devDependencies.chai,
+        'chai-xml': pjson.devDependencies['chai-xml'],
+        'fs-extra': pjson.devDependencies['fs-extra'],
+        mocha: pjson.devDependencies.mocha,
+        supertest: pjson.devDependencies.supertest,
+        xmldoc: pjson.devDependencies.xmldoc,
+        'yeoman-assert': pjson.devDependencies['yeoman-assert']
       },
       author: {
         name: this.props.author,
@@ -850,7 +850,7 @@ module.exports = class extends Generator {
             })
 
           Object.assign(pkgJson.devDependencies, {
-            cypress: JSON.stringify(pjson.devDependencies.cypress)
+            cypress: pjson.devDependencies.cypress
           })
 
           Object.assign(pkgJson.scripts, {

--- a/test/util/consistency.js
+++ b/test/util/consistency.js
@@ -102,7 +102,7 @@ exports.isConsistent = function () {
       const pkg = fs.readFileSync('package.json', 'utf8')
       const parsed = JSON.parse(pkg)
       var pkgDepVer = parsed.devDependencies.mocha
-      var pattern = /\^?\d+\.+/
+      var pattern = /^\^?\d+\.+/
     }
     var devDeps = pattern.test(pkgDepVer)
     assert(devDeps)

